src/dev-server/dev-server-client/polyfills.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-var */
```

Comment:
One last thing - can we add the justification for disabling eslint here?
```suggestion
/* eslint-disable no-var -- shims will continue to use var while we support older browsers */
```